src/Sulu/Bundle/WebsiteBundle/EventSubscriber/AnalyticsSerializeEventSubscriber.php

```diff
@@ -50,5 +50,11 @@ public function onPostSerialize(ObjectEvent $event)
             $value->domains = true;
             $event->getVisitor()->visitProperty($metadata, $value, $event->getContext());
         }
+
+        $content = $analytics->getContent();
```

Comment:
i was also wondering but it seems to happen in combination with this PR but then always. And the problem is, that the metadata will be generated by the first serialization and in the tests there is an array there. this information will be cached and used in the following testcases where it is a string and then the tests are failing.